mlflow/pyfunc/__init__.py

```diff
@@ -923,6 +923,11 @@ def _parse_spark_datatype(datatype: str):
     return udf(lambda x: x, returnType=datatype).returnType
 
 
+def _is_null_array(value):
+    # If provided value is None or NaN, regard it as a null array.
+    return value is None or isinstance(value, float) and np.isnan(value)
```

Comment:
Is `value` an array?

Reply:
yes in normal case.

Reply:
Note Nan is regarded as None, because https://github.com/mlflow/mlflow/issues/8986#issuecomment-1627960699

Reply:
why do we need `isinstance(value, float)`?

Reply:
I'd avoid using `array` because this function seems to work with a non-array value.

Reply:
```
`np.isnan()` will raise error for values that is not number.  We need to add a check before calling `np.isnan()`
```

Reply:
what about the name `_is_null_element`?

Reply:
> `np.isnan()` will raise error for values that is not number. We need to add a check before calling `np.isnan()`

got it, this should be added as a comment

Reply:
can this function only be used to an element in an array?

Reply:
I'd call _is_none_or_nan

Reply:
@WeichenXu123 When naming a function, just describe what the function does. Don't need to assume how it's used.